web/cobrands/sass/_multiselect.scss

```diff
@@ -1,14 +1,15 @@
 .multi-select-container {
     position: relative;
+    display: inline-block; // shrink to width of .multi-select-button
 }
 
 .multi-select-menu {
     position: absolute;
     left: 0;
     top: 0.8em;
-    z-index: 1;
-    float: left;
-    min-width: 100%;
+    z-index: 2;
```

Comment:
Why did this go up?

Reply:
To stack above the footer. I will leave a comment in the sass file.